hw/ip/rv_dm/data/rv_dm_testplan.hjson

```diff
@@ -432,6 +432,36 @@
       stage: V1
       tests: ["rv_dm_dataaddr_rw_access"]
     }
+    {
+      name: halt_resume
+      desc: '''
+            Verify debug module halt/going/resume behavior.
+            - Program jtag_dmi_ral.dmcontrol.haltreq to 1'b1
+            - After a few usec, acknowledge halt by setting tl_mem_ral.halted to 0. (ack for hart index 0).
+            - Send a single valid command with following process:
+              - Set jtag_dtm_ral.dmi.op to dm::DTM_WRITE
```

Comment:
Hm... would it be a bit clearer if we word the writes a little differently, since the CSRs don't actually take on the direct values we write? Instead of "set <reg> to <value>", would "write <value> to <reg>" be a little less confusing?

Reply:
Updated.